### PR TITLE
Extend kgs update / upgrade cluster for CAPI cluster

### PR DESCRIPTION
**Pull request by njuettner**
Towards: https://github.com/giantswarm/roadmap/issues/3638

### CODE REVIEW

**Review by Gacko — APPROVED** [review]

File: src/content/vintage/use-the-api/kubectl-gs/update-cluster.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 linkTitle: update cluster
-title: "'kubectl gs update cluster' command reference"
+title: "'kubectl gs update / upgrade cluster' command reference"
```
Comment (AndiDog) [inline src/content/vintage/use-the-api/kubectl-gs/update-cluster.md:3]: This makes it sound like there's a `kubectl gs upgrade` subcommand. What about
```suggestion
title: "'kubectl gs update cluster' command reference (cluster upgrades)"
```

File: src/content/vintage/use-the-api/kubectl-gs/update-cluster.md [hunk 2]
```diff
@@ -52,9 +52,34 @@ kubectl gs update cluster \
...
+**Note:** This feature is currently only working when the cluster configuration is **not stored** via GitOps.
+
+To upgrade a workload cluster immediately, use the following command:
+
+```nohighlight
+kubectl gs update cluster \
+  --provider capa \
+  --namespace org-acme \
+  --name a1b2c \
+  --release-version 16.1.0
```
Comment (AndiDog) [inline src/content/vintage/use-the-api/kubectl-gs/update-cluster.md:66]:
```suggestion
  --release-version 26.2.0
```

(valid example)

File: src/content/vintage/use-the-api/kubectl-gs/update-cluster.md [hunk 3]
```diff
@@ -52,9 +52,34 @@ kubectl gs update cluster \
...
+```
+
+To schedule a workload cluster upgrade in the future, the `--scheduled-time` flag is used, like in the example below:
```
Comment (AndiDog) [inline src/content/vintage/use-the-api/kubectl-gs/update-cluster.md:69]: Is this already supported?

Comment (njuettner) [reply src/content/vintage/use-the-api/kubectl-gs/update-cluster.md:69]: **Note:** This feature is currently only working when the cluster configuration is **not stored** via GitOps.

I mentioned it above updating cluster is only working when not using GitOps

**Review by marians — COMMENTED** [review]
I'd recommend to move the vintage part to the bottom, below CAPI.

**Comment by marians** [comment]
@njuettner We have just moved the kubectl-gs reference to src/content/reference/kubectl-gs. This means you'll have to adapt your changes to the new location. Sorry about that!

**Comment by njuettner** [comment]
No problem, changed it 👍🏻

**Pull request merged**